pkg/utils/rest/mapper.go

```diff
@@ -65,9 +65,10 @@ func (p *Mapper) Mapper() meta.RESTMapper {
 	return p.mapper
 }
 
-// MappingFor returns the RESTMapping for the Kind given, or the Kind referenced by the resource.
-// Prefers a fully specified GroupVersionResource match. If one is not found, we match on a fully
-// specified GroupVersionKind, or fallback to a match on GroupKind.
+// The resourceOrKindArg should be in the format of {resource}.{version}.{group}, or {resource}.
```

Comment:
@ldpliu Hey quick question. Does `ManagedClusterAction` have the same requirement to use either `{resource}.{version}.{group}, or {resource}` or is this just for the `ManagedClusterView`?

Reply:
it's just for the ManagedClusterView

Reply:
i believe `ManagedClusterAction` also needs `{resource}.{version}.{group}`

Reply:
Sorry for that, Managedclusteraction have the same requirement.